### ISSUE TITLE
Idea: binary releases on GitHub

### ISSUE THREAD

**Issue by alexellis** [alexellis/jaas#25]
How are you using `jaas`? Would it be useful to have a binary release and curl script / brew formula or do you run it inside the Docker container I am providing?

**Comment by dcharbonnier** [comment]
a binary that can be run on a docker alpine container would be nice :-)

**Comment by alexellis** [comment]
Yes that will be possible if we do a static compilation just like with OpenFaaS right now.

**Comment by MarcPer** [comment]
A binary release would be great.

**Comment by aeons** [comment]
Would love this as well. I've tried running the docker image, but it seems that mounting a volume inside the task when running `jaas` inside a docker image has some problems.
